index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" type="image/png" href="./dist/favicon.png" />
+    <title>Markdown Basic Editor</title>
+  </head>
+
+  <body>
+    <script>
+      window.location.replace("/dist/index.html");
```

Comment:
This is..weird. You shouldn't be having a redirect here.

Reply:
I have an index.html in the root directory to redirect to `/dist/index.html` so we don't have to open it `localhost:8004/dist/index.html` every time we start a new dev session. We can just open `localhost:8004`. Do you want me to remove it?

Reply:
Yes you should remove this.

Reply:
Removed 👍